packages/core/src/lib/banner.svelte

```diff
@@ -0,0 +1,146 @@
+<svelte:options immutable />
+
+<script
+  lang="ts"
+  context="module"
+>
+export type BannerVariant = 'danger' | 'warning' | 'success' | 'info';
+</script>
+
+<script lang="ts">
+import cx from 'classnames';
+import { createEventDispatcher } from 'svelte';
+import { Button, Icon } from '$lib';
+
+/** The severity of the notification you want to show users*/
+export let variant: BannerVariant;
+
+/** The scaling applied on the y access of the page*/
```

Comment:
s/access/axis

Reply:
Didn't notice that when I was refactoring, good catch!